src/tests/ftest/pool/evict_test.py

```diff
@@ -86,13 +87,16 @@ def evict_badparam(self, test_param):
             "Pool UUID: %s\n Pool handle: %s\n Server group: %s\n",
             self.pool.uuid, self.pool.pool.handle.value, self.pool.name)
 
-        if test_param == "BAD_SERVER_NAME":
-            # Attempt to evict pool with invalid server group name
-            # set the server group name directly
-            self.pool.pool.group = ctypes.create_string_buffer(test_param)
-            self.log.info(
-                "Evicting pool with invalid Server Group Name: %s", test_param)
-        elif test_param == "invalid_uuid":
+        # Commented out due to DAOS-5545
```

Comment:
In general, commenting out test code to avoid temporarily running a test should be avoided.

In this case however I don't think commenting out is needed at all as these commented out sections of `def evict_badparam(self, test_param)` will never get executed.  This method is called [here](https://github.com/daos-stack/daos/blob/be7081af201ae233ff926bff2eb31e24e794e345/src/tests/ftest/pool/evict_test.py#L285) and [here](https://github.com/daos-stack/daos/blob/be7081af201ae233ff926bff2eb31e24e794e345/src/tests/ftest/pool/evict_test.py#L295).  The first call should be handled by adding `@skipForTicket("DAOS-5545")` to the `def test_evict_bad_server_name(self)` method.  In the second call, the passed in `test_param` will only ever be [`"invalid_uuid"`](https://github.com/daos-stack/daos/blob/master/src/tests/ftest/pool/evict_test.yaml#L23).

Reply:
That is true Phil. Working on this.